<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>codeWindowClass</class>
 <widget class="QMainWindow" name="codeWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qasmGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <widget class="QWidget" name="source">
       <attribute name="title">
        <string>Source</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="listing">
       <attribute name="title">
        <string>Listing</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_13"/>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройка</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Кодировка</string>
     </property>
     <addaction name="action_8"/>
    </widget>
    <addaction name="action_5"/>
    <addaction name="action_TASM"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_QT"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Панели инструментов</string>
     </property>
     <addaction name="action_9"/>
    </widget>
    <addaction name="menu_6"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>Проект</string>
    </property>
    <addaction name="action_tasm"/>
    <addaction name="action_tlink"/>
    <addaction name="action_td"/>
    <addaction name="separator"/>
    <addaction name="action_run"/>
    <addaction name="separator"/>
    <addaction name="action_cansel"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_5"/>
   <addaction name="menu_7"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Параметры подсветки...</string>
   </property>
   <property name="toolTip">
    <string>Параметры подсветки синтаксиса</string>
   </property>
  </action>
  <action name="action_TASM">
   <property name="text">
    <string>Опции TASM...</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>О программе...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_QT">
   <property name="text">
    <string>О библиотеке QT...</string>
   </property>
   <property name="shortcut">
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_8">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>По умолчанию</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="action_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Основная панель инструментов</string>
   </property>
  </action>
  <action name="action_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Строка статуса</string>
   </property>
  </action>
  <action name="action_tasm">
   <property name="text">
    <string>Компиляция</string>
   </property>
  </action>
  <action name="action_tlink">
   <property name="text">
    <string>Линковка</string>
   </property>
  </action>
  <action name="action_td">
   <property name="text">
    <string>Отладка</string>
   </property>
  </action>
  <action name="action_run">
   <property name="text">
    <string>Запуск</string>
   </property>
  </action>
  <action name="action_cansel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Отмена операции</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Увеличить</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>Уменьшить</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Показать листинг</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
